### ISSUE TITLE
Browser language is not considered/respected

### ISSUE THREAD

**Issue by kelson42** [kiwix/web#158]
My language browser configuration is French (privileged content language). But if I type https://kiwix.org in the browser, this is the English version which is delivered https://www.kiwix.org/en/. It should be https://www.kiwix.org/fr/

**Comment by martin8032** [comment]
To my knowledge, the automatic redirection to a specific language has negative implications on the Google index:

https://wpml.org/documentation/getting-started-guide/language-setup/automatic-redirect-based-on-browser-language/how-browser-language-redirect-affects-google-indexing/

**Comment by kelson42** [comment]
Primary I'm cobcerned it works fine for users. If google does bul****, we should not follow it on this.

**Comment by martin8032** [comment]
There is no reason to blame Google in this case. If you install a redirect, the Google Bot is redirected as well (to the EN version, that is, missing out the other languages).

**Comment by kelson42** [comment]
To fix the problem, what should be done in Wordpress?

**Comment by martin8032** [comment]
Citing the source mentioned above: «This is related to how Google works and not to how the language redirect is implemented in WPML.»

**Comment by kelson42** [comment]
I don't mean Google, this is not my job. I mean this ticket.

**Comment by martin8032** [comment]
I disagree. Making sure that Google is indexing all of our language versions is much more important, especially if we have the strategic goal to reach more people outside the English-speaking countries. (And the redirection makes even less sense since our translations are far from complete and up-to-date.

**Comment by kelson42** [comment]
Google will index things properly, my guess. We have such a system since 10 years on wiki.kiwix.org and nobodies has ever complained. I made a brief check with two browsers in different languages and all looks good.

The link you have given seems pretty old and the author basically says that he has no clue how Google search engine works. He just not want to be blamed if google does bul****.

**Comment by martin8032** [comment]
The information may be old, but according to my experience it's absolutely correct. We have actived the language redirect on the W****** website recently, and as a consequence the German language version was removed from the Google index. I will not make the same mistake again.

**Comment by kelson42** [comment]
Situation should be reassessed with new Web site, I believe it has been fixed. @rgaudin to confirm
